src/constants.ts

```diff
@@ -12,15 +13,24 @@ export const INFO_FILE_NAME = 'info'
 export const SCHEMA_FILE_NAME = 'schema_init'
 export const PG_DIFF_CONFIG_FILE_NAME = 'pg-diff-config'
 export const PG_SCHEMA_DIFF_FILE_NAME = 'schema_diff'
+export const PREFERENCES_FILE_NAME = config.preferencesFileName
 
 export const ROOT_FOLDER = path.join(getAppEntryPointDir(), '../')
 export const DATABASE_FOLDER = path.join(getAppEntryPointDir(), config.databaseFolder)
 export const SNAPSHOT_FOLDER = path.join(DATABASE_FOLDER, SNAPSHOT_SUBFOLDER)
 export const SNAPSHOT_OPTIONS_FOLDER = path.join(DATABASE_FOLDER, OPTIONS_SUBFOLDER)
```

Comment:
This file is still a bit of a mess and I'm not super happy with how we've ended up with multiple "config" type files, but it's too big a job to do a complete overhaul right now, so just sticking with current pattern here.